### PR TITLE
Data tier

### PR DESCRIPTION
**Pull request by mpeamma**

### CODE REVIEW

File: datatier.tf [hunk 1]
```diff
@@ -1 +1,70 @@
...
+  skip_final_snapshot     = true                                          #NOTE: This option is set to true only for testing. Not to be used in production
+  backup_retention_period = 5
+  publicly_accessible     = false
+  vpc_security_group_ids  = ["${aws_security_group.db-sg.id}"]
+  port                    = "5432"
+  identifier              = "master-db"
+  db_subnet_group_name    = "${aws_db_subnet_group.db-subnet-group.name}"
+}
+
+resource "aws_db_instance" "replica-db" {
```
Comment (sean-nixon) [inline datatier.tf]: I've not worked with PostgreSQL in RDS before so could you help explain what this is doing? Is this a read replica? Can the main DB instance not be made multi-AZ?

File: variables.tf [hunk 2]
```diff
@@ -97,11 +97,11 @@ apt install -y git
 git clone https://github.com/mpeamma/todo-api-node
 cd todo-api-node
 npm install
-npm start &
+DB_HOST=%s npm start &
```
Comment (sean-nixon) [inline variables.tf]: Curious how you're handling the password for the DB connection. Is that not implemented yet?

Comment (mpeamma) [reply variables.tf]: Whoops! Forgot to add that in there. My little test node app has the same defaults for passwords. I'll update it

File: datatier.tf (hunk 1 above)
Comment (mpeamma) [reply datatier.tf]: I may have misread the network diagram. I read it as wanting a master DB and a read replica that was multi AZ

Comment (sean-nixon) [reply datatier.tf]: I can see how you'd arrive at that. We were going for a single RDS instance that's multi-AZ. The second RDS icon indicates the secondary server that exists under the hood in a multi-AZ deployment

Comment (mpeamma) [reply datatier.tf]: Sounds good. I'll remove the replica and make the master multi_az

**Pull request merged**